packages/pakistan.provinces/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@svg-maps/pakistan.provinces",
```

Comment:
Do you think the package should be called `paskitan.administrative-units` to be more clear/consistent? 🤔I don't know if people really use this official name or just call them "provinces" .